tests/unit/acl.tcl

```diff
@@ -482,25 +482,18 @@ start_server {tags {"acl external:skip"}} {
     }
 
     test "ACL CAT without category - list all categories" {
-        set reply [r acl cat]
-        assert_match "*keyspace*" $reply
-        assert_match "*connection*" $reply
-    }
-
-    test "ACL CAT category - list all commands that belong to category" {
-        # We simply just check a few categories.
-        assert_match "*llen*" [r ACL CAT list]
-        assert_match "*zcard*" [r ACL CAT sortedset]
-        assert_match "*exists*" [r ACL CAT keyspace]
-        assert_match "*flushall*" [r ACL CAT dangerous]
+        set categories [split [string trim [r acl cat] "\r\n"]]
```

Comment:
why do we need the `split`? it returns an array, which tcl converts to a list.
i see that in other tests below you didn't use it... why here?

Reply:
weird, last night, before the `split` or the `trim`, the `lsearch` can not lookup the element.
so i thought it is some kind of tcl things...
now it is ok. i must be not very awake before going to bed and do something wrong.